apis/proto/v1/vald/remove.proto

```
@@ -26,28 +26,134 @@ option java_multiple_files = true;
 option java_outer_classname = "ValdRemove";
 option java_package = "org.vdaas.vald.api.v1.vald";
 
-// Remove service provides ways to remove indexed vectors.
+// Overview
+// Remove Service is responsible for removing vectors indexed in the `vald-agent`.
 service Remove {
```

buf (1.47.2)

Service name "Remove" should be suffixed with "Service".

(SERVICE_SUFFIX)